lib/Rex/Commands/File.pm

```diff
@@ -314,7 +314,7 @@ Furthermore, if a path prefix matches multiple prefix entries in 'path_map',
 e.g. "files/etc/ntpd.conf" matching both "files/" and "files/etc/", the
 longer matching prefix(es) have precedence over shorter ones. Note that
 keys without a trailing slash (i.e. "files/etc") will be treated as having
-a trailing slash when matching the prefix ("files/etc/"). 
+a trailing slash when matching the prefix ("files/etc/").
```

Comment:
OK, this trailing space removal is needed, but out of the scope of the topic at hadn (creating target directory before exatracting stuff), so if possible I'd put it into a separate commit.